big_tests/tests/metrics_session_SUITE.erl

```diff
@@ -113,16 +111,23 @@ auth_failed(Config) ->
 session_global(Config) ->
     escalus:story(Config, [{alice, 1}], fun(_Alice) ->
 
-        timer:sleep(?GLOBAL_WAIT_TIME),
-        assert_counter(global, 1, totalSessionCount)
+        wait_for_counter(global, 1, totalSessionCount)
 
         end).
 
 session_unique(Config) ->
     escalus:story(Config, [{alice, 2}], fun(_Alice1, _Alice2) ->
 
-        timer:sleep(?GLOBAL_WAIT_TIME),
-        assert_counter(global, 1, uniqueSessionCount),
-        assert_counter(global, 2, totalSessionCount)
+        wait_for_counter(global, 1, uniqueSessionCount),
+        wait_for_counter(global, 2, totalSessionCount)
 
         end).
+
+wait_for_counter(Host, ExpectedValue, Counter) ->
```

Comment:
Can't we extract a common part out of this definition and [this one](https://github.com/esl/MongooseIM/pull/1982/files#diff-e51d3c9fb9c07615eb6d1ded14daa8e5R270). As far as I see, they differ only in this `?WAIT_TIME` value.

Reply:
The difference is in `Host` parameter, but anyway, common part can be extracted